_episodes/10-diagnostics.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "Writing your own diagnostic script"
+teaching: TBD
+exercises: TBD
+
+questions:
+- "How to read preprocessor output from a Python diagnsotics?"
+- "How to write a new diagnostic in ESMValTool?"
+
+objectives:
+- "Understanding the interface between the ESMValCore preprocessor and a diagnostic script."
+- "Writing a new Python diagnostic"
+
+keypoints:
+- "Take home message 1"
+- "etc ..."
+---
+
+## Introduction
+
+The diagnostic script is an important component of ESMValTool where the actual 
```

Comment:
```suggestion
The diagnostic script is an important component of ESMValTool where the
```

Reply:
I would leave this out, as quite a lot of "science" is done in the preprocessors now.